control-plane-operator/controllers/hostedcontrolplane/assets/olm/catalog-community.cronjob.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: community-operators-catalog
+  annotations:
+    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+spec:
+  concurrencyPolicy: Forbid
+  schedule: '* */1 * * *'
+  startingDeadlineSeconds: 200
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      backoffLimit: 2
+      activeDeadlineSeconds: 600
+      template:
+        spec:
+          serviceAccountName: catalog-restarter-pods
+          restartPolicy: Never
+          containers:
+          - name: kubectl
+            image: bitnami/kubectl
```

Comment:
Unsure whether using this `kubectl` image is appropriate, and whether a redhat registry image exists.

Reply:
you can use the `cli` image from the release payload